.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+
+[**]
+end_of_line = lf
```

Comment:
Add `charset = utf-8` here rather than python block

Reply:
sry but did not get u

Reply:
instead of [**] should i write charset = utf-8 ?

Reply:
No, @newbazz Add a line `charset = utf-8`.

Reply:
@mohitkyadav okk thanks.

Reply:
I made the changes.